rust-runtime/aws-smithy-http-server/src/request/connect_info.rs

```diff
@@ -48,12 +50,11 @@ use tower_http::add_extension::{AddExtension, AddExtensionLayer};
 
 use crate::{request::FromParts, Extension};
 
-/// A [`MakeService`] created from a router.
+/// A [`MakeService`] used to insert [`ConnectInfo<T>`] into [`http::Request`]s.
 ///
-/// See [`Router::into_make_service_with_connect_info`] for more details.
```

Comment:
This intradoc link would be possible with https://github.com/awslabs/smithy-rs/issues/2014